.eslintrc.json

```diff
@@ -113,6 +113,7 @@
         "switch-colon-spacing": "error",
         "template-curly-spacing": ["error", "never"],
         "unicode-bom": ["error", "never"],
-        "yoda": "error"
+        "yoda": "error",
+        "curly": ["error", "all"]
```

Comment:
can you put the rule in alphabetical order?

Reply:
sure